tests/acceptance/DownloadGigaDBTemplateFiles.feature

```diff
@@ -0,0 +1,22 @@
+Feature: An author can download file template
+  As an author
+  I want to download upload form template after login
+  So that I can upload data using the form
+
+  Background: Sign is as a user
+    Given I sign in gigadb website as "user"
```

Comment:
This will need changing if the `iSignInAsAUser()` step is used from AuthorSteps.php.